Monkey Patch/Player/Behavior.cpp

```diff
@@ -158,6 +158,12 @@ CMultiPatch CMPatches_SR1Reloading = {
 		CMPatches_UseWeaponAfterEmpty.Apply();
 	}
 
+	void HigherMaxSpeed()
+	{
+		Logger::TypedLog(CHN_MOD, "Patching Higher Max Speed...\n");
+		patchFloat((BYTE*)0x00DD0768, 446.59299);
```

Comment:
```suggestion
		patchNop((void*)0x00AEB541, 2);
		patchNop((void*)(0x00AEB541 + 0x61), 2);
```
Instead of out-right changing this float address which is used in other functions such as havok related/player stuff, it'd make sense to just stop the vehicle parsing from limiting the max speed outright.